forge/ee/routes/flowTemplates/index.js

```diff
@@ -0,0 +1,238 @@
+const { registerPermissions } = require('../../../lib/permissions')
+const { Roles } = require('../../../lib/roles.js')
+
+const hasValueChanged = (requestProp, existingProp) => (requestProp !== undefined && existingProp !== requestProp)
+
+module.exports = async function (app) {
+    registerPermissions({
+        'flow-template:create': { description: 'Create a Flow Template', role: Roles.Admin },
+        'flow-template:list': { description: 'List all Flow Templates' },
+        'flow-template:read': { description: 'View a Flow Template' },
+        'flow-template:delete': { description: 'Delete a Flow Template', role: Roles.Admin },
+        'flow-template:edit': { description: 'Edit a Flow Template', role: Roles.Admin }
+    })
+
+    app.get('/', {
+        preHandler: app.needsPermission('flow-template:list'),
+        schema: {
+            summary: 'Get a list of the available flow templates',
+            tags: ['Flow Templates'],
+            query: { $ref: 'PaginationParams' },
+            response: {
+                200: {
+                    type: 'object',
+                    properties: {
+                        meta: { $ref: 'PaginationMeta' },
+                        count: { type: 'number' },
+                        templates: { $ref: 'FlowTemplateSummaryList' }
+                    }
+                },
+                '4xx': {
+                    $ref: 'APIError'
+                }
+            }
+        }
+    }, async (request, reply) => {
+        const paginationOptions = app.getPaginationOptions(request)
+        let filter = { active: true }
+        if (request.query.filter === 'all') {
+            filter = {}
+        } else if (request.query.filter === 'active') {
+            // Default behaviour
```

Comment:
Should this just be an else? It's potentially confusing to have to parse the comment and line 37 to see what's happening

Reply:
This was a copy-n-paste from another route that applies the same filtering logic, so I didn't spend too much thought on it. Happy to live with it for now.